adapters/Vungle/VungleAdapter/VungleRouter.m

```diff
@@ -157,8 +169,9 @@ - (BOOL)hasDelegateForPlacementID:(NSString *)placementID
   return result;
 }
 
-- (BOOL)canRequestBannerAdForPlacementID:(NSString *)placmentID {
-  return self.mrecPlacementID == nil || [self.mrecPlacementID isEqualToString:placmentID];
+- (BOOL)canRequestBannerAdForPlacementID:(NSString *)placmentID withBannerType:(VungleNetworkAdapterAdType)bannerType{
+    NSString *placement = (bannerType == MREC ? _mrecPlacementID : _bannerPlacementID);
+    return placement == nil || [placement isEqualToString:placmentID];
```

Comment:
`[placement isEqualToString:placmentID]` should be `![placement isEqualToString:placmentID]` ?

We probably implemented this as only one banner at a time. But this will be wrong.
We'd like to allow multiple banners with different placement IDs to load. (Not play).
This will prevent pubs from preparing multiple banners cached. 

@crbinsf Do you remember why we needed to regulate only one MREC at a time?

Reply:
I remembered why we needed to do this here.
Since "banner" ad type doesn't have "load and cached" state (a pub always call only "getBanner" and it will load and play sequentially), so we needed to stop requesting at this point if another banner is "playing".